picard/options.py

```diff
@@ -0,0 +1,350 @@
+import logging
+
+from PyQt6 import QtCore
+
+from picard.config import (
+    BoolOption,
+    FloatOption,
+    IntOption,
+    ListOption,
+    Option,
+    TextOption,
+)
+from picard.const import MUSICBRAINZ_SERVERS
+from picard.const.defaults import (
+    DEFAULT_AUTOBACKUP_DIRECTORY,
+    DEFAULT_CA_PROVIDERS,
+    DEFAULT_CAA_IMAGE_SIZE,
+    DEFAULT_CAA_IMAGE_TYPE_EXCLUDE,
+    DEFAULT_CAA_IMAGE_TYPE_INCLUDE,
+    DEFAULT_CACHE_SIZE_IN_BYTES,
+    DEFAULT_COVER_IMAGE_FILENAME,
+    DEFAULT_CURRENT_BROWSER_PATH,
+    DEFAULT_DRIVES,
+    DEFAULT_FPCALC_THREADS,
+    DEFAULT_LOCAL_COVER_ART_REGEX,
+    DEFAULT_LONG_PATHS,
+    DEFAULT_MUSIC_DIR,
+    DEFAULT_PROGRAM_UPDATE_LEVEL,
+    DEFAULT_QUERY_LIMIT,
+    DEFAULT_RELEASE_TYPE_SCORES,
+    DEFAULT_REPLACEMENT,
+    DEFAULT_SHOW_MENU_ICONS,
+    DEFAULT_STARTING_DIR,
+    DEFAULT_THEME_NAME,
+    DEFAULT_TOOLBAR_LAYOUT,
+    DEFAULT_TOP_TAGS,
+    DEFAULT_WIN_COMPAT_REPLACEMENTS,
+)
+from picard.i18n import N_
+
+from picard.ui.colors import InterfaceColors
+
+
+# picard/coverart/providers/caa.py
+# Cover Art Archive Cover Art Archive: Release
```

Comment:
Possible wording for instructions.

```suggestion
from picard.ui.colors import InterfaceColors


# Note: There are two steps to adding an option to an option page:
#
#   1. The option is added to the appropriate section below.
#
#   2. The option is registered in the __init__ method of the option
#      page with a call to the page's `register_setting()` method.

# picard/coverart/providers/caa.py
# Cover Art Archive Cover Art Archive: Release
```

Reply:
On the second point, I would rephrase it a bit since that's an optional step, only if a Option in section 'setting' has a matching way to edit it.

Reply:
Good point.  I've updated my suggestion to try to cover that.

Reply:
@rdswift let's work on that after I merge this elephant PR.